### DISCUSSION TITLE
404 errors

### DISCUSSION THREAD

**Discussion by mediaexplosion** [DSJAS/DSJAS#89] [Installation help]
I am having issues with 404 Errors on all of the extra pages. The main accounts work, but the fraud contact get started and apply all go to 404 errors. This happens on all themes.

**Comment by ejv2** [comment]
Please make sure that AllowOverride is set to all in your main apache configuration and try again.

**Comment by mediaexplosion** [comment]
I'm running NGINX I have been trying to work thru alternative fixes and have not had any luck so far. Everything else seems to work fine.

**Reply by ejv2** [reply]
The site relies almost entirely on the Apache rewrite engine to function, so using nginx is not currently supported.

**Comment by mediaexplosion** [comment]
I switched the webserver to Apache from NGINX, but now all i get are 500 errors. Nothing else changed.  Any tips for where to look?

**Reply by ejv2** [reply]
Try having a look in the apache access logs  and see what exactly caused the 500 error.

**Reply by mediaexplosion** [reply]
OK I tracked it down to the htaccess files
```
# Force output buffering (fixes header errors)
# php_value output_buffering 4096
```
If I comment out the php_value line it will load and I have to comment out the following to access the admin panel
```
<Files ~ "installTheme.php$">
# php_value upload_max_filesize 50M
# php_value post_max_size 42M
</Files>

# Allow larger uploads when installing modules
<Files ~ "installTheme.php$">
# php_value upload_max_filesize 50M
# php_value post_max_size 42M
```
I'm running PHP 8.1 and Apache 2.4
I'm also running into a weird databse errors with the transfer module telling me mysqli_stmt object is already closed in..

**Reply by ejv2** [reply]
So it seems that every line with ``php_value`` seems to cause a 500... strange.

Would you mind posting the original error log so that I can see what the issue is specifically? It might be too big to post here. If so, make a Pastebin or email it to me. Thanks!

**Reply by ejv2** [reply]
For anybody with a similar issue, this appears to be caused by using PHP through CGI rather than linked directly as an Apache module. This causes 500 errors for any part of the site with an htaccess file which attempts to configure PHP using php_value.

This isn't conclusive yet (we haven't fully resolved the issue) but if it is, I will mark this as the answer.
